doc/usage.md

```diff
@@ -1,54 +1,56 @@
 # Usage
 
-1. Import FHCookieGuard
+## 1. Import FHCookieGuard
 
 ```javascript
-   import FHCookieGuard from '@freshheads/cookie-guard';
+import FHCookieGuard from '@freshheads/cookie-guard';
 ```
 
-2. Initialize
+## 2. Initialize
 
 ```javascript
 new FHCookieGuard('.js-cookie-alert', {
-   // optional config options 
+   // Optional config options 
 });
 ```
 
-3. Set notification template near end of body tag
+## 3. Set notification template near end of body tag
 
 ```twig
-{% set excludedPages = [ path('cookies') ] %}
-
-<div class="cookie-alert js-cookie-alert" data-excluded-page-urls="{{ excludedPages|json_encode }}" role="alertdialog" aria-labelledby="cookie-alert-title" aria-describedby="cookie-alert-description" aria-hidden="true">
-    <div class="cookie-alert__content">
-        <div class="cookie-alert__header">
-            <h2 class="cookie-alert__title" id="cookie-alert-title">Cookie usage</h2>
-        </div>
-        <p class="cookie-alert__description" id="cookie-alert-description">
-            We use cookies to analyse the website, so we can further improve the user experience. Can we use cookies?
-            <a href="{{ path('cookies'}) }}" class="cookie-alert__link" target="_blank">Read more information</a> about how we use cookies.
-        </p>
-        <div class="cookie-alert__actions">
-            <button class="button button--secondary js-cookie-alert-accept">Yes, continue</button>
-            <button class="button button--link js-cookie-alert-refuse">No, continue without cookies</button>
+{% if not cookie_settings_submitted() %}
```

Comment:
al is dit natuurlijk alleen icm met de bundle

Reply:
Denk dat die kans bijna 100% is als je ook Twig gebruikt?

Reply:
ja alleen wordt de bundle pas eronder benoemd, terwijl er al wel functionaliteiten ervan boven in de example staan. Maar denk dat het indd wel altijd samen gebruikt wordt.